docs/faq.md

```diff
@@ -81,7 +81,7 @@ We list some common troubles faced by many users and their corresponding solutio
 
 - "RuntimeError: Expected to have finished reduction in the prior iteration before starting a new one"
     1. This error indicates that your module has parameters that were not used in producing loss. This phenomenon may be caused by running different branches in your code in DDP mode.
-    2. You can set ` find_unused_parameters = True` in the config to solve the above problems or find those unused parameters manually.
+    2. You can set ` find_unused_parameters = True` in the config to solve the above problems(but this will slow down the training speed)  or find those unused parameters with `detect_anomalous_params=True` in `OptimizerHook` then delete them.
```

Comment:
Put the description of `detect_anomalous_params=True` as a third point and indicate the minimum MMCV version requirement.